JSDemos/Demos/DataGrid/RecordGrouping/description.md

```diff
@@ -1 +1,9 @@
-The **DataGrid** widget allows you and your end users to&nbsp;group data against an&nbsp;unlimited number of&nbsp;columns. In&nbsp;this example, data is&nbsp;grouped by&nbsp;the _Country_ column (a&nbsp;group index is&nbsp;specified for this column in&nbsp;the **columns** array). Your users can group data by&nbsp;dragging column headers to&nbsp;the group panel. You can control its visibility using the **groupPanel**&nbsp;| **visible** option. A&nbsp;user can also reorder the columns using drag and drop, because the **allowColumnReordering** option is&nbsp;set to _true_.
\ No newline at end of file
+The **DataGrid** supports data grouping by a column or multiple columns. 
```

Comment:
I'd like to update this to avoid the -ing form ("grouping"), if possible. I also think it's better to mention "users", since they are the ones who will be performing these actions. For example:

`The **DataGrid** allows users to group data by a single column or multiple columns.`

Can we say something like this instead?